### PR TITLE
Night deprecation

### PR DESCRIPTION
**Pull request by prazi**
#### Changelog notes 

```
* Added/Extended operations....
```

#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: distributor-api-v1/operations.md [hunk 1]
```diff
@@ -421,11 +421,11 @@ Alternative initial call used to obtain all static data about hotel relevant for
...
                     }
-                },
-                "Charging": "PerRoomNight",
-                "Posting": "Daily",
-                "Ordering": 1
-            }
+                }
+            },
+            "ChargingMode": "PerTimeUnit",
+            "PostingMode": "Daily",
```
Comment (koprivajakub) [inline distributor-api-v1/operations.md:427]: image

`PostingMode` example should be `PerTimeUnit` right?

Same thing for line 137 https://github.com/MewsSystems/gitbook-distributor-guide/pull/60/files#diff-e0d8177d43ad27101a34daf5ee1404fa5b293c8884f331abcbe852ac6671a819R137

**Review by koprivajakub — CHANGES_REQUESTED** [review]
We need to firstly document the Services data... My recommendation is to document that part first with a changelog notice and then do the deprecation. This way the customer of the API would not know what PerTimeUnit means

File: distributor-api-v1/changelog.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 08.03.2021
+
+* Added `AverageAmountPerTimeUnit` field to [Room price](./operations.md#room-price) replacing the `AverageAmountPerNight`
+* Added `ChargingMode` field to [Product](./operations.md#product) replacing the `Charging`
+* Added `PostingMode` field to [Product](./operations.md#product) replacing the `Posting`
+* Added `SettlementMaximumTimeUnits` field to [Rate group](./operations.md#rate-group) replacing the `SettlementMaximumNights`
+
 ## 21.07.2021
```
Comment (prazi) [inline distributor-api-v1/changelog.md:10]: also I realized I forgot to update the date when the PR was merged so we have the creation date in the changelog, should I change it to 08.02.2021 now? @koprivajakub

**Review by koprivajakub — APPROVED** [review]

File: distributor-api-v1/deprecations/average-amount-per-night.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Average amount per night deprecation
+`AverageAmountPerNight` field of [room price](../operations.md#room-price) object is deprecated and will be removed from the response of [hotels/getAvailability](../operations.md#get-availability) and [reservations/getPricing](../operations.md#get-reservations-pricing).. It is replaced by `AverageAmountPerTimeUnit`.
```
Comment (vitasamek) [inline distributor-api-v1/deprecations/average-amount-per-night.md:2]: missing empty line.

Comment (vitasamek) [reply distributor-api-v1/deprecations/average-amount-per-night.md:2]: double dots `pricing)..`

File: distributor-api-v1/deprecations/product-charging.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Product charging deprecation
+`Charging` field of [product](../operations.md#product) object is deprecated and will be removed from the response of [hotels/get](../operations.md#get-hotels) and [configurations/get](../operations.md#get-configuration).. It is replaced by [Product charging mode](../operations.md#product-charging-mode).
```
Comment (vitasamek) [inline distributor-api-v1/deprecations/product-charging.md:2]: missing empty line

Comment (vitasamek) [reply distributor-api-v1/deprecations/product-charging.md:2]: again double dots at the end.

File: distributor-api-v1/deprecations/product-posting.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Product posting deprecation
+`Posting` field of [product](../operations.md#product) object is deprecated and will be removed from the response of [hotels/get](../operations.md#get-hotels) and [configurations/get](../operations.md#get-configuration).. It is replaced by [Product posting mode](../operations.md#product-posting-mode).
```
Comment (vitasamek) [inline distributor-api-v1/deprecations/product-posting.md:2]: The same

File: distributor-api-v1/deprecations/settlement-maximum-nights.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# SettlementMaximumNights deprecation
+`SettlementMaximumNights` field of [rate group](../operations.md#rate-group) object is deprecated and will be removed from the response of [hotels/getAvailability](../operations.md#get-availability). It is replaced by `SettlementMaximumTimeUnits`.
```
Comment (vitasamek) [inline distributor-api-v1/deprecations/settlement-maximum-nights.md:2]: Empty line

**Pull request merged**